GitUI/CommandsDialogs/SettingsDialog/Pages/GitConfigSettingsPage.Designer.cs

```diff
@@ -555,20 +560,43 @@ private void InitializeComponent()
             this.tableLayoutPanelGitConfig.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanelGitConfig.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanelGitConfig.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelGitConfig.RowStyles.Add(new System.Windows.Forms.RowStyle(System.Windows.Forms.SizeType.Absolute, 25F));
             this.tableLayoutPanelGitConfig.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanelGitConfig.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanelGitConfig.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanelGitConfig.RowStyles.Add(new System.Windows.Forms.RowStyle(System.Windows.Forms.SizeType.Percent, 100F));
-            this.tableLayoutPanelGitConfig.Size = new System.Drawing.Size(1310, 778);
+            this.tableLayoutPanelGitConfig.Size = new System.Drawing.Size(913, 778);
             this.tableLayoutPanelGitConfig.TabIndex = 81;
             // 
+            // DiffWarnLabel
+            // 
+            this.DiffWarnLabel.Anchor = System.Windows.Forms.AnchorStyles.Left;
+            this.DiffWarnLabel.AutoSize = true;
+            this.DiffWarnLabel.Location = new System.Drawing.Point(3, 285);
+            this.DiffWarnLabel.Name = "DiffWarnLabel";
+            this.DiffWarnLabel.Size = new System.Drawing.Size(51, 13);
+            this.DiffWarnLabel.TabIndex = 80;
+            this.DiffWarnLabel.Text = "Diff warn";
```

Comment:
Please do not use abbreviations. They are very difficult to translate into other languages